CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Changelog
 
 ## Unreleased
-- Updated `django`, `flask`, `httplib`, `requests` and `pyramid` modules.
+- Updated `django`, `flask`, `httplib`, `requests` and `pyramid` modules
   ([#755](https://github.com/census-instrumentation/opencensus-python/pull/755))
 - Added `http code` to `grpc code` status code mapping on `utils`
   ([#746](https://github.com/census-instrumentation/opencensus-python/pull/746))
+- Updated `requests` module
+  ([#771](https://github.com/census-instrumentation/opencensus-python/pull/771))
```

Comment:
Do we need to update the CHANGELOG for the core sdk?

Reply:
Is just for tracking for the next release, so we don’t forget to update some module. I remember someone suggested this recently, but I can’t remember in which PR. It does not need to go into the release changelog (in my opinion), but this way we know what needs to be updated !

Reply:
I see. As long as we are consistent, I feel like it should be fine. So far it seems like we only update the changelog if there are changes in the actual package but your argument for it being a "release reminder" is valid :).